actions/ql/src/change-notes/2025-02-06-curate-suites.md

```diff
@@ -0,0 +1,17 @@
+---
+category: queryMetadata
```

Comment:
Can we indicate somehow this is breaking and that alerts from the moved queries will close for existing users?

Reply:
I've changed the category to `breaking` and enhanced the description.